.k8s/base/coursemapper-kg/concept-map/deployment.yaml

```
@@ -32,6 +32,9 @@ spec:
           value: postgres://postgres:password@$(WP_PG_SERVICE):5432
         - name: WIKIPEDIA_FALLBACK
           value: "false"
+        envFrom:
+        - configMapRef:
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 10 but found 8

(indentation)